src/renderer/components/output.tsx

```diff
@@ -96,6 +101,10 @@ export class Output extends React.Component<CommandsProps> {
     }
   }
 
+  public getLineNumber(originalLineNumber: number) {
+    return this.outputTimeStamps[originalLineNumber - 1];
```

Comment:
Right! I assumed since we initialize our output with "Output ready 🔬 " we would always have something there..just realized that that was a horrible choice from a maintainable standpoint 😔